### PR TITLE
es: Course length mistake

### PR DESCRIPTION
**Pull request by salvamunoz**
Edit course length from 3 days to 4 which is actually the total length

### CODE REVIEW

**Comment by deavid** [comment]
The original english text does not mention the length. I would recommend to just rearrange the translation to match the same.

Also, you need to sign the CL as mentioned above.

**Comment by mgeisler** [comment]
@salvamunoz, can you take out the mentioning of days like @deavid suggests? Then we can merge this!

File: po/es.po [hunk 1]
```diff
@@ -1217,7 +1217,7 @@ msgid ""
 "course covers the full spectrum of Rust, from basic syntax to advanced "
 "topics like generics and error handling."
 msgstr ""
-"Este es un curso de Rust de tres días que ha desarrollado el equipo de "
+"Este es un curso de Rust de cuatro días que ha desarrollado el equipo de "
```
Comment (mgeisler) [inline po/es.po:1220]: @salvamunoz and @deavid, is this correct Spanish?

```suggestion
"Este es un curso de Rust que ha desarrollado el equipo de "
```

**Comment by deavid** [comment]
Yes, that's correct Spanish

File: po/es.po (hunk 1 above)
Comment (mgeisler) [reply po/es.po:1220]: Pushed via command line commit.

**Comment by mgeisler** [comment]
> Yes, that's correct Spanish

Thanks a lot! :tada:

**Review by mgeisler — APPROVED** [review]

**Pull request merged**